### PR TITLE
[15.0][MIG] maintenance_plan

### PR DESCRIPTION
**Pull request by BernatPForgeFlow**

### CODE REVIEW

**Comment by BernatPForgeFlow** [comment]
@ForgeFlow 
@LoisRForgeFlow 
@AaronHForgeFlow

File: maintenance_plan/models/maintenance_equipment.py [hunk 1]
```diff
@@ -13,17 +13,17 @@ class MaintenanceEquipment(models.Model):
     _inherit = "maintenance.equipment"
 
     maintenance_plan_ids = fields.One2many(
-        string="Maintenance plan",
+        name="Maintenance plan",
```
Comment (LoisRForgeFlow) [inline maintenance_plan/models/maintenance_equipment.py]: This is the string of the name, that was correct, the name is better to be left with the default value.

File: maintenance_plan/models/maintenance_kind.py [hunk 2]
```diff
@@ -10,7 +10,7 @@ class MaintenanceKind(models.Model):
     _name = "maintenance.kind"
     _description = "Maintenance Kind"
 
-    name = fields.Char("Name", required=True, translate=True)
+    name = fields.Char(name="Name", required=True, translate=True)
```
Comment (LoisRForgeFlow) [inline maintenance_plan/models/maintenance_kind.py]: string

**Review by AaronHForgeFlow — APPROVED** [review]
Functional tests :+1:

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM, thanks!

**Review by etobella — APPROVED** [review]

**Pull request merged**